stories/MegaNav.stories.js

```diff
@@ -88,9 +88,17 @@ const Template = (args) => ({
                                 :key="subHeadingIndex"
                                 :list-heading="subHeading.title"
                             >
-                                <template 
-                                    v-slot:nav-list-items
-                                >
+                                <template v-slot:nav-heading-cta-link>
```

Comment:
@duncansanderson is this a breaking change? does this need to be done on each of the projects when this is released?